src/main/java/org/wildfly/security/ssl/SSLConfiguratorImpl.java

```diff
@@ -68,8 +68,8 @@
     }
 
     void configure(SSLParameters params, String[] supportedProtocols, String[] supportedCipherSuites) {
-        params.setProtocols(protocolSelector.evaluate(supportedProtocols));
-        params.setCipherSuites(cipherSuiteSelector.evaluate(supportedCipherSuites));
+        if (supportedProtocols != null) params.setProtocols(protocolSelector.evaluate(supportedProtocols));
```

Comment:
I think this might be better with an Assert,checkNotNull check - in what situations could the supported protocols and cipher suites be null?